ast/compile.go

```diff
@@ -1737,7 +1798,9 @@ func (c *Compiler) resolveAllRefs() {
 }
 
 func (c *Compiler) removeImports() {
+	c.imports = make(map[string][]*Import, len(c.Modules))
 	for name := range c.Modules {
+		c.imports[name] = c.Modules[name].Imports
```

Comment:
I couldn't find the reason for why we're stripping the imports out. I stopped looking after I found https://github.com/tsandall/opa/commit/c6226621c493bf0b4da4499ec8c899ceac82d3bb from @johanfylling. @johanfylling do you recall why we're doing  this?

Reply:
It looks like this was previously part of the `compile_stage_resolve_refs` stage, and was probably moved into its own stage because the `compile_stage_check_duplicate_imports` stage still needed the imports to be present. If I disable the `compile_stage_remove_imports` stage, I see failing tests, so later stages seems to rely on the imports being removed ..